import "goby/moos/protobuf/frontseat_config.proto";
import "goby/common/protobuf/option_extensions.proto";

message WavegliderSV2FrontSeatConfig
{
    required string pm_serial_port = 1;
    optional uint32 pm_serial_baud = 2 [default = 115200];
    optional uint32 board_id = 3 [default = 0x03];
    optional uint32 task_id = 4 [default = 0x01];
}

extend iFrontSeatConfig
{
    optional WavegliderSV2FrontSeatConfig waveglider_sv2_config = 1001;
}
